tutorials/commands/job-submit.rst

```diff
@@ -0,0 +1,55 @@
+.. _job-submit:
+
+==========================
+How to Submit Jobs in Flux
+==========================
+
+Do you want to submit a job to Flux? Here's a short tutorial on how to do so!
+
+----------------------------
+Submit Jobs Using Flux's CLI
+----------------------------
+
+From within a Flux instance, you can submit your job on the command line with
+optional arguments, resource options, per task options, and per resource
+options:
+
+.. code-block:: sh
+
+    $ flux mini submit --nodes=2 --ntasks=4 --cores-per-task=2 ./my_compute_script.lua 120
+    ƒM5k8m7m
+    $ flux mini submit --nodes=1 --ntasks=1 --cores-per-task=2 ./my_other_script.lua 120
+    ƒSUEFPDH
+
+A jobID is returned for every job submitted. You can view the status of your
+running jobs with ``flux jobs``:
+
+.. code-block:: sh
+
+    $ flux jobs
+       JOBID  USER     NAME       ST NTASKS NNODES     TIME INFO
+    ƒSUEFPDH  fluxuser my_other_s  R      1      1   1.842s 
+    ƒM5k8m7m  fluxuser my_compute  R      4      2   3.255s
+
+You can also get information about your job by passing in the jobID and a key.
+For example, to get resource information about your job:
+
+.. code-block:: sh
+
+  $ flux job info ƒM5k8m7m R
```

Comment:
Is there a way to get this in a table (non json) - this was a question I had the other day and I couldn't figure out from the command line.

Reply:
I think actually we should not advertise `flux job info` in a high-level tutorial. It is more of a "plumbing" command. Instead we should focus on `flux jobs`, the main interface users will use to get information about their jobs.

Reply:
Oops! Sorry for including it in this tutorial. I'll go ahead and just remove this section then and only include `flux jobs`.